Source/stores.h

```diff
@@ -12,7 +12,7 @@ extern BYTE *pSTextBoxCels;
 extern int premiumlevel;
 extern int talker;
 extern STextStruct stext[24];
-extern char stextsize;
+extern BOOLEAN stextsize;

```

Comment:
`stextsize` actually isn't a boolean, even though it's basically used as such. From the symbol file we have:
```
Def class EXT type CHAR size 0 name stextsize
Def class EXT type UCHAR size 0 name stextscrl
```
Since the PSX SDK (Psy-Q) contained neither BOOLEAN/BOOL (these are Microsoft types) they typedef'd them as a `UCHAR`. So all bools/booleans will be unsigned.

Reply:
I did an update to the header file, but forgot that this also needs to be updated in the source file. So, `stores.cpp` should be updated before merging this PR.